app/models/user.rb

```diff
@@ -2,6 +2,7 @@ class User < ApplicationRecord
   include DeviseTokenAuth::Concerns::User
 
   has_many :rents, dependent: :destroy
+  has_many :books, dependent: :destroy
```

Comment:
What I meant here is that a `Book` entity could have many `rents`, but this is fine. The only thing you are missing is the `through` option. You need to specify from where does the user obtain the books, in this case it's through rents.
Try it out via rails console creating a User, a book, some rents and then doing something like `user.books` to see if it works or not.

Also you can populate the `seeds.rb` file so you don't have to create those examples over and over again to test

Reply:
That's True :man_facepalming: Sorry!